### PR TITLE
Flink version is updated to 1.11.2 and consul-api is updated to 1.4.5

### PR DESCRIPTION
**Pull request by paulh86**

### CODE REVIEW

**Comment by kbialek** [comment]
@paulh86 Thank you for the contribution. Are you actually using flink-consul for your workloads?

File: src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java [hunk 1]
```diff
@@ -78,4 +79,10 @@ public void removeJobGraph(JobID jobId) throws Exception {
 	private String path(JobID jobID) {
 		return jobgraphsPath + jobID.toString();
 	}
+
+    @Override
+    public void releaseJobGraph(JobID jobId) throws Exception {
+        // TODO Auto-generated method stub
```
Comment (kbialek) [inline src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java]: Is the implementation missing?

Comment (paulh86) [reply src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java]: Good Question! I will check it, if i can keep it with an empty implementation and therefore simply remove the autogenerated comment. Sry.

**Comment by paulh86** [comment]
> @paulh86 Thank you for the contribution. Are you actually using flink-consul for your workloads?

Yes, i try to use it in our environment. We are running on nomad with consul as service registration and discovery. I will test it in a few days :)

File: src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java (hunk 1 above)
Comment (paulh86) [reply src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java]: I checked the method. There is no need for proper implementing this, only added a comment.

**Comment by paulh86** [comment]
@kbialek i will also add another pull-request for providing a consul url with optional tls configuration. Or should i implement the logic in this pull-request?

**Comment by kbialek** [comment]
@paulh86 
> Yes, i try to use it in our environment.

That's awesome. Please let me know if it works. Actually for me that is only a pet project which I created to experiment with flink and consul few years ago, so I'm very curious how does it behave real environment.

> i will also add another pull-request for providing a consul url with optional tls configuration

I think PR should not mix features, so please open a new one.

**Pull request merged**